test/unit/mdc-floating-label/mdc-floating-label.test.js

```diff
@@ -21,7 +21,7 @@ import {assert} from 'chai';
 import {MDCFloatingLabel} from '../../../packages/mdc-floating-label/index';
 
 const getFixture = () => bel`
-  <div class="mdc-floating-label"></div>
+  <label class="mdc-floating-label"></label>
```

Comment:
I think this change snuck in here

Reply:
This is done on purpose. You're right in that it is more related to the floating label PR, but it wasn't caught until doing changes with line-ripple.